tests/cli/test_cli_download.py

```diff
@@ -13,6 +13,7 @@
 )
 
 
+@pytest.skip("Skip this test.", allow_module_level=True)
```

Comment:
Why do we have to skip this test?

Reply:
Because I didn't want to fix the mocked API client. I've fixed that and re-enabled this anyway.

Reply:
Awesome thanks!